client/src/time-tracker/time-tracker.actions.js

```diff
@@ -26,6 +26,7 @@ export const handleTimerStart = time => {
         type: UPDATE_START_TIME,
         time,
     }
+    // add more
```

Comment:
non-blocking: this comment was used already, it would be nice to be more creative